User_Story#1/SortingApp/ViewModel/CollectionSortingViewModel.cs

```diff
@@ -153,6 +153,7 @@ private void ReadArray(object obj)
 
                 foreach (var number in this.handler.UnSortedCollection)
                     this.UnSortedCollectionOfNumbers.Add(number);
+
                 this.Message = "Array was successfully read.";
             }
             catch (Exception e)
```

Comment:
If you had a dependency injection for VM, you would create tests for VM too.
So just to be clear: VM should be covered for Unit tests too.
But I believe it will be module testing, anyway it is a part of unit tests)